<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>brwWindow</class>
 <widget class="QMainWindow" name="brwWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>681</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Borrow/Return</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Card Number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editCno"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="cardQueryBtn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Query</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>BookNumber:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editBno"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="borrowBtn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Borrow</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>281</width>
       <height>171</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Corresponding Book Number:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="stateText">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>150</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="retBtn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>80</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(F)</string>
    </property>
   </widget>
   <addaction name="menuFile_F"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
